### ISSUE TITLE
Ammo count and oxygen are not rendered

### ISSUE THREAD

**Issue by Lyberta** [ptitSeb/Serious-Engine#7]
Single color HUD elements such as ammo count bars and oxygen bars are not rendered.

OS: Debian Testing amd64
GPU: Radeon RX 590
Driver: Mesa 18

**Comment by ptitSeb** [comment]
Oh strange. Can you upload a screenshot? I'm pretty sure I have seen ammo count working at some point.

**Comment by Lyberta** [comment]
![Screenshot at 2019-03-11 03-03-15](https://user-images.githubusercontent.com/10873185/54093576-63d64980-43aa-11e9-9221-3133b70d45a0.png)

**Comment by ptitSeb** [comment]
![preview3](https://user-images.githubusercontent.com/3250792/54158877-da775380-444b-11e9-9965-7da0ea78e7a8.png)
You mean, the little bar in the shell (and other type ammo) icon on the bottom right of the screen, like with SeriousSam First Encounter?

**Comment by Lyberta** [comment]
Yes.

**Comment by ptitSeb** [comment]
So, I think the Oxygen drawing, for Second Encounter, is done line 1074-1087 of `Sources/EntitiesMP/HUD.cpp`. And for the ammos, probably a bit above. I'll try to debug the code, to see why it's not always drawn (I have some old screenshot with some ammos drawn, but not all of them).

**Comment by parkerlreed** [comment]
Not seeing an oxygen counter 
![image](https://user-images.githubusercontent.com/841440/63215548-3a0e1500-c0f6-11e9-892c-f8a2d39ae5fb.png)

**Comment by parkerlreed** [comment]
Oh I just wasnt there long enough :D 
![image](https://user-images.githubusercontent.com/841440/63215579-9cffac00-c0f6-11e9-81d8-8195ef1a2461.png)

**Comment by ptitSeb** [comment]
Yep, you need to be underwater for some time before the counter appears. Serious Sam has some serious lungs !
